.github/pull_request_template.md

```diff
@@ -1,10 +1,12 @@
 # Description
 
-> Detailed changes summary | testing | dependencies
+> Required; detailed changes summary | testing | dependencies
 > Good example: https://os.mbed.com/docs/latest/reference/guidelines.html#workflow (Pull request template)
 
 # Pull request type
 
+> Required; Please tick one of the following types
```

Comment:
Why not use actual comments?
<!--- looky a comment! --->
https://stackoverflow.com/a/39737760

Reply:
Good, let me test

Reply:
@geky because we want to be able to automate the checking of the PR type and automatically assign the release tag